packages/ketcher-polymer-editor-react/src/components/menu/menuItem/multiItem/MultiItem.tsx

```diff
@@ -0,0 +1,122 @@
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+import { MenuItemVariant } from 'components/menu/menu.types'
+import { useState } from 'react'
+import Icon from 'components/shared/ui/Icon/Icon'
+import { ClickAwayListener } from '@mui/material'
+import { SingleItem } from 'components/menu/menuItem/singleItem/SingleItem'
+import styled from '@emotion/styled'
+import Collapse from '@mui/material/Collapse'
+
+interface MultiItemPropType {
+  options: MenuItemVariant[]
+  onClick: (name: MenuItemVariant) => any
+  activeTool: MenuItemVariant
+  vertical?: boolean
+}
+
+const MultiItemContainer = styled('div')`
+  display: flex;
+  position: relative;
+`
+
+const OptionsFlexContainer = styled('div')`
+  display: flex;
+  position: absolute;
+  background-color: white;
+  left: 5px;
+  border-radius: 2px;
+  flex-direction: ${(props) => (props['data-orientation'] ? 'column' : 'row')};
+`
+
+const DropDownIcon = styled(Icon)`
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`
+
+const OptionsItemsCollapse = styled(Collapse)`
+  position: relative;
+`
+
+const MultiItemHeader = styled('div')`
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 28px;
+  height: 28px;
+  margin: 4px 0;
+  padding: 0;
+  justify-content: center;
+  border-radius: 2px;
+  background-color: ${(props) =>
+    props['data-active'] ? 'rgba(0, 131, 143, 0.4)' : 'white'};
+
+  :hover {
+    transform: scale(1.2);
+  }
+`
+
+const MultiItem = ({
+  options,
+  activeTool,
+  onClick,
+  vertical
+}: MultiItemPropType) => {
+  const [open, setOpen] = useState(false)
+
+  const handleDropDownClick = () => {
+    setOpen((prev) => !prev)
+  }
+
+  const headerMultiTool = options.includes(activeTool) ? activeTool : options[0]
+  const isActiveTool = activeTool === headerMultiTool
+
+  return (
+    <MultiItemContainer>
+      <MultiItemHeader
```

Comment:
TypeScript seems to ignore `data-*` attributes here, which allows for flexibility, but maybe let's make it more type-safe? We'll avoid errors and take advantage of intellisense.

We can extend a styled component by annotation:

```tsx
// Make extended type for a div
type MultiItemHeaderProps = {
  isActive: boolean
} & React.HTMLAttributes<HTMLDivElement>

// annotate new component returned by styled()
const MultiItemHeader = styled('div')<MultiItemHeaderProps>`
  position: relative;
  width: 28px;
  height: 28px;
  margin: 4px 0;
  padding: 0;
  justify-content: center;
  border-radius: 2px;
  background-color: ${(props) =>
    // use our new prop
    props.isActive ? 'rgba(0, 131, 143, 0.4)' : 'white'};

  :hover {
    transform: scale(1.2);
  }
`
```
And pass value for `isActive` as for a regular React prop:
```tsx
...
<MultiItemHeader
    isActive={isActiveTool}
    onClick={() => onClick(headerMultiTool)}>
      ...
 </MultiItemHeader>
...
```